init.org

```diff
@@ -4547,7 +4567,29 @@ Packages that I am currently testing or evaluating.
 #+begin_src emacs-lisp :tangle yes
 (use-package mermaid-mode
   :ensure t
-  :mode "\\.\\(mmd|mermaid\\)\\'")
+  :mode "\\.\\(mmd|mermaid\\)\\'"
+  :commands
+  (mermaid-compile-buffer)
+  :preface
+  (defun mermaid-mode-set-compile-command ()
+    "Configure compile command for mermaid-mode."
+    (set (make-local-variable 'compile-command)
+         (string-join (append (list (shell-quote-argument mermaid-mmdc-location)
```

Comment:
You could use `(mapconcat #'shell-quote-argument (list ...) " ")` here.

Reply:
Thank you, I updated to use this convention, that was nicer.